deepchecks/core/checks.py

```diff
@@ -160,6 +167,62 @@ def metadata(self, with_doc_link: bool = False) -> CheckMetadata:
             summary=get_docs_summary(self, with_doc_link)
         )
 
+    def config(self, with_module=True) -> CheckConfig:
+        """Return check config (conditions' config not yet supported).
+
+        Parameters
+        ----------
+        with_module : bool, default True
```

Comment:
when we will set it to False?

Reply:
if we're doing it for our own suites we won't need the module

Reply:
as it'll be set in the suite

Reply:
but why do you care about having it twice to simplify the code?